src/Forms/Select.js

```diff
@@ -39,12 +78,37 @@ const ArrowSelect = styled.div`
   transform: translateY(-50%);
 `;
 
-export default () => (
-  <Select>
-    <Text>1 пассажир,&nbsp;</Text>
-    <TextLight>эконом</TextLight>
-    <ArrowSelect>
-      <Icon icon="arrowSelect" />
-    </ArrowSelect>
-  </Select>
-);
+class Select extends Component {
+  state = {
+    active: false
+  };
+
+  handleClick = () => {
+    this.setState({
+      active: !this.state.active
```

Comment:
https://medium.freecodecamp.org/functional-setstate-is-the-future-of-react-374f30401b6b